### PR TITLE
Added delay to tomb spawn

### LINKED ISSUE #1980: MvP Tomb Spawntime
"On official servers MvP Tombs spawn a few seconds after the MvP is killed.

On rathena (Hercules) the MvP Tomb spawns immediately. So sometimes it's kinda hard to loot dropped Items near the tomb."

By Jeybla.

Link: https://github.com/rathena/rathena/issues/1429

### PR DESCRIPTION
**Pull request by guilherme-gm**
[//]: # (**********************************)
[//]: # (** Fill in the following fields **)
[//]: # (**********************************)

[//]: # (Note: Lines beginning with syntax such as this one, are comments and will not be visible in your report!)

### Pull Request Prelude

[//]: # (Thank you for working on improving Hercules!)

[//]: # (Please complete these steps and check the following boxes by putting an `x` inside the brackets _before_ filing your Pull Request.)

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR will be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

This adds a configurable delay time to spawning a tomb after a MVP is killed.

[//]: # (Describe at length, the changes that this pull request makes.)

**Affected Branches:** 

[//]: # (Master? Slave?)

master

**Issues addressed:**

Fixes #1980 

[//]: # (Issue Tracker Number if any.)

### Known Issues and TODO List

[//]: # (Insert checklist here)
[//]: # (Syntax: - [ ] Checkbox)

[//]: # (**NOTE** Enable the setting "[√] Allow edits from maintainers." when creating your pull request if you have not already enabled it.)

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

**Comment by HerculesWSAPI** [comment]
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/2001)

File: src/map/mob.c [hunk 1]
```diff
@@ -157,6 +157,42 @@ int mobdb_searchname_array_sub(struct mob_db* monster, const char *str, int flag
 /*==========================================
  *              MvP Tomb [GreenBox]
  *------------------------------------------*/
+ /// Creates a timer to spawn a tomb
+ /// @param nd : The tomb
+void mvptomb_spawn_delayed(struct npc_data *nd)
+{
+	if (nd->u.tomb.spawn_timer != INVALID_TIMER)
```
Comment (4144) [inline src/map/mob.c]: missing nullpo check for nd

File: conf/map/battle/monster.conf [hunk 2]
```diff
@@ -234,6 +234,10 @@ mob_slave_keep_target: true
 // See http://irowiki.org/wiki/MVP#Gravestone
 mvp_tomb_enabled: true
 
+// Delay before a tomp is spawned, in milliseconds.
```
Comment (Jedzkie) [inline conf/map/battle/monster.conf]: `Tomb`

**Comment by guilherme-gm** [comment]
@MishimaHaruna fixed. Thanks!

**Pull request merged**